scripts/qa_crowbarsetup.sh

```diff
@@ -1814,13 +1804,13 @@ function set_proposalvars()
     fi
 
     # Cinder
-    if [[ ! $cinder_conf_volume_type ]] ; then
+    if [[ -z "$cinder_backend" ]] ; then
```

Comment:
No need for -z and quotes here - its in double backets.
It was part of a consistency cleanup of this function.
I even thought about doing it for the entire script later.